app/app/layout.tsx

```diff
@@ -0,0 +1,25 @@
+import "../styles/global.scss";
+import AuthProvider from "@/contexts/Auth";
+import SidebarLayout from "@/components/SideBarLayout";
+
+/**
+ * The root layout for the whole app
+ * 
+ * todo: metadata
+ */
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    // <Html lang="en" style={{fontSize: "14px"}} data-bs-theme="dark">
+    <html lang="en" style={{ fontSize: "14px" }}>
```

Comment:
the default bootstrap font size is `16`, and i think the ACC designs go down to `12`, maybe smaller. it's an accessibility issue that needs more consideration 👀

Reply:
Ah, thanks pointing to MUI's approach—interesting that they default to 14px. 

My understanding of the way it works is that `1rem` = whatever the browser default is, unless specified in the `html` tag, in which case `1rem` becomes the specified value.